contracts/contracts/zapper/WOETHCCIPZapper.sol

```diff
@@ -190,6 +190,6 @@ contract WOETHCCIPZapper {
         emit Zap(messageId, msg.sender, receiver, amount);
 
         // Return the message ID
-        return messageId;
+        return (messageId, woethReceived);
```

Comment:
no need to explicitly return. Any return variables assigned in the function body like `messageId` and `woethReceived` will be returned even if the return is not called.

FYI, in older Solidity compilers it used to be less gas to not explicitly return, but that's no longer the case with newer Solidity compilers

Reply:
Should uint256 be removed from line 156?

Reply:
yes it should